src/components/WebexAvatar/WebexAvatar.jsx

```diff
@@ -43,8 +47,11 @@ export default function WebexAvatar({
   style,
 }) {
   const {avatar, displayName, status} = usePerson(personID);
+  const [randomColor] = useState(avatarColors[Math.floor(Math.random() * avatarColors.length)]);
```

Comment:
you may use a function as an argument to useState to avoid recalculating the initial state on each render

https://reactjs.org/docs/hooks-faq.html#how-to-create-expensive-objects-lazily